app/routes/task.py

```diff
@@ -0,0 +1,143 @@
+from flask import Blueprint, jsonify, make_response, request
+from app import db
+from app.models.task import Task
+from datetime import datetime
+import os
+from dotenv import load_dotenv
+import requests
+
+task_bp = Blueprint('tasks', __name__, url_prefix='/tasks')
+
+@task_bp.route('', methods=['GET'])
+def read_all_tasks():
+    tasks = db.session.query(Task).filter(Task.accessible==True)
+    
+    if request.args.get('sort') == 'asc':
+        tasks = tasks.order_by(Task.title.asc())
+    elif request.args.get('sort') == 'desc':
+        tasks = tasks.order_by(Task.title.desc())
+
+    response_body = [task.to_dict() for task in tasks]
+    return make_response(jsonify(response_body), 200)
+
+@task_bp.route('', methods=['POST'])
+def create_new_task():
+    request_body = request.get_json()
+    if 'title' not in request_body or 'description' not in request_body or 'completed_at' not in request_body:
+        response_body = {
+            'details': 'Invalid data'
+        }
+        return make_response(jsonify(response_body), 400)
+    # goal = None if 'goal_id' not in request_body else request_body['goal_id']
+    task_to_create = Task(
+        title=request_body['title'],
+        description=request_body['description'],
+        completed_at=request_body['completed_at']
+        # goal_id=
+    )
```

Comment:
An alternative here would be to delete the if-block and instead use a try/except that looked for any key errors on the `request_body`.